### DISCUSSION TITLE
[SOLVED my bad] Hello looking for help how this looks on IMDb

### DISCUSSION THREAD

**Discussion by JayBe13** [Purfview/IMDb-Scout-Mod#139] [Ideas (implemented)]
First I have to say Thank you SO much for this. I love it!

I was wondering if you or someone here can help me? I have scoured the settings and searched but I can't seem to find it.
Right now this is what IMDb looks like to me.

![ImdbScoutMod1](https://user-images.githubusercontent.com/98782558/198905311-dc88f9ec-8927-45fd-913c-ced40dff81bd.jpg)

I would like to look like this.

![ImdbScoutMod2](https://user-images.githubusercontent.com/98782558/198905340-b9c9379e-9874-42ad-af31-3f1b8abf6c7c.jpg)

Am I overlooking a setting or would I have to edit your script to make it custom to have it look the way I want? 

Thank you.

**Comment by Purfview** [comment]
As I understand you don't want to use any sites but ratings, in v17.7 I added two new options at Title page section's bottom.

**Reply by JayBe13** [reply]
TY for your reply I am Sorry it took me so long to reply I have been away and have to leave again tonight :( I have version 17.8 now and this is what I see 
![image](https://user-images.githubusercontent.com/98782558/202054346-18f20330-b379-4f08-9be5-3aa71d2bb108.png)
what I am looking to do is removing this part 
![image](https://user-images.githubusercontent.com/98782558/202054474-ad2a8e0b-3aa1-4742-9c8a-fa772aaf04b2.png)

I hope that is a bit more clear what I was trying to say.

**Comment by Purfview** [comment]
I understand what you wrote.
Do you understand what I answered? Enable those two new options to get what you want.

**Reply by JayBe13** [reply]
Sorry for late reply I was out of town. I feel like an idiot!! I was looking for the settings you mention by going to extensions clicking Extension options on Tapermonkey, clicking on the installed userscripts tab clicking on edit for IMDb Scout Mod and clicking the settings tab there. Instead of just clicking on Tapermonkey opening IMDb Scout Mod Settings and checking Disable the searchable sites code.

![facepalm](https://user-images.githubusercontent.com/98782558/206905207-1f7269d9-5a00-48d7-bdee-ca1a73212516.png)

**Comment by Purfview** [comment]
LOL, there is the "Load Settings" button in your screenshots...
